### DISCUSSION TITLE
A feature to use the normal windows explorer to import mods again

### DISCUSSION THREAD

**Discussion by noTMesb** [xivdev/Penumbra#206] [Ideas]
The new mod importer doesnt allow me to access my mods anymore over a network shared device where they are stored so was wondering if an option to use the new or old explorers to import would be good

**Comment by SerenitySublime** [comment]
Not going to happen as it was a pain on the dev side having the windows explorer option. This is discussed at length in the Penumbra Discord. Feel free to join to read more into it.

**Reply by SerenitySublime** [reply]
Also, feel free to leave feedback for the new selection option in goat place Discord as it is a Dalamud item that Otter is using for Penumbra.
